### DISCUSSION TITLE
Can't connect BH1750, help!

### DISCUSSION THREAD

**Discussion by ooomakon** [RavenSystem/esp-homekit-devices#1627] [Q&A]
home does not connect to the program
{
  "c": { "l": 13 },
  "a": [{
    "t": 50,
    "n": 2,
	"dt":[5,4,35],
    "f": 14357564.33,
    "po": 1.396066767,
    "re": 8550
  }]
}

**Comment by ooomakon** [comment]
With such parameters, it is also not detected, for pairing with Homekit
{"c":{"o":0,"l":13},"a":[{"t":50,"n":2,"dt":[5,4,35]}]}

**Reply by homelink21** [reply]
I have the same problem with this configuration, BH1750 sensor connected to the Wemos in the following scheme (VCC -> 3.3v - SCL -> D1 [GPIO5] - SDA -> D2 [GPIO4] - GND -> G) and the following json: { "c": {"l": 14, "ct": 10, "ic": [5,4,400], "n": "ESP32_PRL", "sn": "PIR32L"}, "a": [{ "t": 50, "n": 2, "dt": [4,92]}, {"t": 12, "i": 10, "f1": [{"g": 0}]}]} .
In addition I have configured the i2c bus "ic" in the general configuration.

**Reply by ooomakon** [reply]
Works!
{"c":{"ic":[[5,4,400]]},"a":[{"t":50,"n":2,"dt":[0,35]}]}

**Reply by Andreadf67** [reply]
on my shelly1 dont work.
{"c":{"ic":[[3,1,400]]},"a":[{"t":50,"n":2,"dt":[0,35]}]}

I use  (VCC -> 3.3v - SCL -> D1 [GPIO3] - SDA -> D2 [GPIO1] - GND -> G) 
Why do you use 400 in "ic" value ?

**Reply by Andreadf67** [reply]
Great, I tried again and it works.
thank you so much

**Comment by ooomakon** [comment]
Tried all json options, it is not added to the house
